### DISCUSSION TITLE
who can help me?

### DISCUSSION THREAD

**Discussion by GerGitHub** [filecoin-project/lotus#10655] [Ideas]
### Checklist

- [X] This is **not** brainstorming ideas. If you have an idea you'd like to discuss, please open a new discussion on [the lotus forum](https://github.com/filecoin-project/lotus/discussions/categories/ideas) and select the category as `Ideas`.
- [X] I **have** a specific, actionable, and well motivated feature request to propose.

### Lotus component

- [ ] lotus daemon - chain sync
- [ ] lotus fvm/fevm - Lotus FVM and FEVM interactions
- [ ] lotus miner/worker - sealing
- [ ] lotus miner - proving(WindowPoSt/WinningPoSt)
- [ ] lotus JSON-RPC API
- [ ] lotus message management (mpool)
- [X] Other

### What is the motivation behind this feature request? Is your feature request related to a problem? Please describe.

My ImToken wallet was stolen and 635 FILs were stolen, please help us ban transactions from the other party's FIL address

My FIL address: f1i4jvh46duxvw3lr43rgxogiptcj6il3j3zi3bba.
The other party's FIL address: f17nkgfc67s3jji7a2yanmqwfvkfqewdpytgw24tq
ID: f02091478

The other party stole the wallets of multiple people in more than 10 minutes;

https://filscan.io/address/general?address=f17nkgfc67s3jji7a2yanmqwfvkfqewdpytgw24tq

### Describe the solution you'd like

please help us! thanks

### Describe alternatives you've considered

please help us ban transactions from the other party's FIL address
The other party's FIL address: f17nkgfc67s3jji7a2yanmqwfvkfqewdpytgw24tq
ID: f02091478

### Additional context

![image](https://user-images.githubusercontent.com/20123237/227794349-34bf778f-635e-40e1-9080-9f555cabef70.png)
![image](https://user-images.githubusercontent.com/20123237/227794387-b171b8e6-43b1-4d54-a565-ec78136f59db.png)
![test 1](https://user-images.githubusercontent.com/20123237/227794414-d1b5c975-139a-47bc-80bd-537ca799a4ab.png)

![image](https://user-images.githubusercontent.com/20123237/227794568-761f0393-90ef-4fbe-a2eb-2a7aead70968.png)
![image](https://user-images.githubusercontent.com/20123237/227794507-8ac0b77c-5340-419c-85d7-30eb48a0c7c9.png)

**Comment by Derek-zd** [comment]
No one can help you with this kind of thing.
